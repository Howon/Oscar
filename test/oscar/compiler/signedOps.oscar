def main() => unit = {
  println(5 + -5);
  println(5 * -5);
  println(5 / -5);
  println(5 - -5);
}