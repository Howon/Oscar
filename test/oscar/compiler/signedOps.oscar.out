0
-25
-1
10